cmd/ui/src/components/GraphButtons/GraphButtons.tsx

```diff
@@ -91,4 +107,61 @@ const GraphButtons: FC<GraphButtonsProps> = ({ rankDirection, options, nonLayout
     );
 };
 
+const GraphMenu: FC<{ label: string; children: ReactNode }> = ({ children, label }) => {
```

Comment:
I think the `GraphMenu` component and the `exportToJson` functionality would be good candidates to move to the `bh-shared-ui` library, so that they don't need to be duplicated for enterprise

Reply:
gotcha. moved both of these!